spt/OrangeBox/modules/ClientDLL.cpp

```diff
@@ -415,6 +421,36 @@ void ClientDLL::ResetAfterframesQueue()
 
 void ClientDLL::OnFrame()
 {
+#if !defined(OE)
+	extern int GetEntityCount();
+	extern CBaseEntity* GetEntity(int);
+
+	if (y_spt_piwsave.GetString()[0] != '\0')
+	{
+		int count = GetEntityCount();
+		for (int i = 0; i < count; ++i)
```

Comment:
Are you sure going through all entities every frame is a good idea? Perhaps there’s some function that we can hook that is called when something gets stuck in the wall.